containers/ddev-webserver/ddev-webserver-base-files/usr/local/bin/n-install.sh

```diff
@@ -31,28 +31,29 @@ fi
 
 ln -sf "/mnt/ddev-global-cache/n_prefix/${HOSTNAME}" "${N_PREFIX}"
 
-# try normal install that also uses cache
+# try online install that also uses cache
 n_install_result=true
-timeout 30 n install "${N_INSTALL_VERSION}" 2> >(tee /tmp/n-install-stderr.txt >&2) || n_install_result=false
+log-stderr.sh --timeout 30 n install "${N_INSTALL_VERSION}" || n_install_result=false
```

Comment:
I wonder if our scattered 30-second timeouts are a general mistake, or if we should figure out a way to parameterize them. No need to change anything right now, just food for thought.